src/TestSuite/FixtureSchemaExtension.php

```diff
@@ -34,12 +37,29 @@ public function __construct()
     {
         FixtureLoader::setInstance(new FixtureDataManager());
 
-        // This isn't a great place for this, but I plan on revisiting
-        // how DB logging works in tests soon.
         $enableLogging = in_array('--debug', $_SERVER['argv'] ?? [], true);
+        $this->configureLogging($enableLogging);
         $this->aliasConnections($enableLogging);
     }
 
+    /**
+     * Configures a logger for SQL queries.
+     *
+     * @param bool $enableLogging Whether or not logging should be enabled.
+     * @return void
+     */
+    protected function configureLogging(bool $enableLogging): void
```

Comment:
Think this should be enableLogging. Calling `configure` with false that just early returns seems odd.